Machine Learning/Outlier Detection/Introduction/README.md

```diff
@@ -0,0 +1,6 @@
+# advent-outliers
+
+To run this example, please ensure that you have the following:
+
+* an elasticsearch cluster running on `localhost:9200`
```

Comment:
This will only work against 7.X (whichever is the release that has outlier detection - sorry forgot which one it was), right?